src/cli/bin.js

```diff
@@ -2,6 +2,7 @@
 
 'use strict'
 
+require('pipe-args').load()
```

Comment:
Yeah, but it's not only add that supports stdin. You can find a list of the commands that supports stdin and which argument it applies to here: https://github.com/ipfs/go-ipfs/search?utf8=%E2%9C%93&q=EnableStdin

We should probably have something similar for js-ipfs.